src/test/java/net/dancier/dancer/chat/ChatControllerTest.java

```diff
@@ -113,7 +116,6 @@ void postChatShouldReturnTheChat() throws Exception {
         }
 
         @Test
-        @WithUserDetails("user@dancier.net")
```

Comment:
This annotation makes no sense. Why?
It lead to a call to the Userdetails service to load an Principal into the Security Context.
But this happens before the @BeforeEach method!
For the reason the populated Authentication Object will not have a dancer id.

An Authentication Object will be created later, when you perform the HTTP Call and hand in the cookie.
Then the filter will go again to the Userdetails service. But now after the before each method.
So it will have an dancer-id.
@jans510

Reply:
ah yes, I had forgotten to remove it when I added the cookie in the test. First the test was running only with this annotation and without cookie but then the dancer id was moved to the authenticated user so the cookie was needed to have all data needed for the test.